interpreter/text/arrange.ml

```diff
@@ -205,11 +205,15 @@ let rmw = function
   | I32 I32Op.RmwXchg | I64 I64Op.RmwXchg -> "xchg"
   | _ -> assert false
 
-let memop name {ty; align; offset; _} =
-  value_type ty ^ "." ^ name ^
+let memop_without_type name {ty; align; offset; _} =
```

Comment:
Nit: can remove `ty` in pattern.

Reply:
`ty` is used below when writing alignment. Maybe the name makes this unclear -- it could be `memop_without_type_prefix` perhaps?

Reply:
Ah, sorry, missed that. I think it's fine as is then.